teuthology/task/ansible.py

```diff
@@ -66,6 +66,8 @@ class Ansible(Task):
                     again during teardown with a 'cleanup' var set to True.
                     This will allow the playbook to clean up after itself,
                     if the playbook supports this feature.
+        reconnect:  If set to True (the default), then reconnect to hosts after
```

Comment:
we need a mention of why one might want this; specifically something like "The playbook may change targets' sshd config, so we want the new config to take effect."